scripts/write-configs.js

```diff
@@ -20,13 +20,55 @@ if (!process.env.KANGAROO_DEV) {
     );
 }
 
+if (kangarooConfig.bootstrapUrl === 'https://dev-test-bootstrap2.holochain.org/') {
```

Comment:
Added some prominent warnings if the dev-test servers are being used. Happy to get feedback on wording.